src/test/java/tictactoe/ControllerTest.java

```diff
@@ -1,14 +1,34 @@
 package tictactoe;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 import java.io.*;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
 import java.util.Arrays;
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class ControllerTest {
 
+    @BeforeEach
+    void clearTable() {
+        var database = new Database();
+        var query = "DELETE FROM saved_games";
+
+        try {
+            var conn = database.connect();
+            PreparedStatement preparedStatement = conn.prepareStatement(query);
```

Comment:
I see this piece of logic being repeated - I'd suggest wrapping it into a method.